packages/core/src/fhir-to-cda/cda-templates/components/__tests__/shared.ts

```diff
@@ -0,0 +1,35 @@
+import { Bundle } from "@medplum/fhirtypes";
+import dayjs from "dayjs";
+import { formatDateToCdaTimestamp, formatDateToHumanReadableFormat } from "../../commons";
+
+export function createEmptyBundle(): Bundle {
+  return {
+    resourceType: "Bundle",
+    type: "collection",
+    entry: [],
+  };
+}
+
+export function getPastDateInDifferentFormats(subtractYears?: number): {
+  dateFhir: string;
+  dateXml: string;
+  dateHumanReadable: string;
```

Comment:
After reading the function, I wonder if it wouldn't be better to be explicit about what the functions return:
- dateFhir > dateIso
- dateXml > dateCda

Reply:
Done